yb-voyager/cmd/config.go

```diff
@@ -397,6 +338,11 @@ func bindCobraFlagsToViper(cmd *cobra.Command, v *viper.Viper) error {
 				bindErr = err
 				return
 			}
+			overrides = append(overrides, ConfigFlagOverride{
+				FlagName:  f.Name,
+				ConfigKey: configKeyPrefix + "." + f.Name,
```

Comment:
nit: use variable for configKey because it is also used above in `v.GetString` (for all cases)